_posts/2021-09-16-data-prepper-roadmap.markdown

```diff
@@ -8,45 +8,46 @@ date:   2021-09-16 01:01:01 -0700
 
-Today, Data Prepper 1.0 supports Trace Analytics data processing as shown below in Figure 2 below, and includes support for OpenTelemetry collector as a source, a raw-trace processor for trace data processing, and a service-map processor for service map creation.
+Data Prepper supports trace analytics data processing, as shown in Figure 2, and includes support for OpenTelemetry collector as a source, a raw-trace processor for trace data processing, and a service-map processor for service map creation.
 
 ![Data Prepper Basics](/assets/media/blog-images/2021-09-16-data-prepper-roadmap/figure2.png){: .img-fluid}
 
-*Figure 2: Trace Analytics data processing pipeline*
+*Figure 2: Trace analytics data processing pipeline*
 
 ### Roadmap
 
-A [roadmap](https://github.com/opensearch-project/data-prepper/projects/1) for Data Prepper was published in the Data Prepper OpenSearch project repository on GitHub. It will give you visibility on plans for new features, enhancements, and bug fixes. We look forward to your feedback on it. In the next few months, the focus is on log collection from multiple sources through Data Prepper where they can be ingested, viewed, and analyzed in OpenSearch Dashboards. Data Prepper 1.2 (December 2021) release is going to provide users the ability to send logs from Fluent Bit to OpenSearch or Amazon OpenSearch Service and use Grok to enhance the logs. These logs can then be correlated to traces coming from the OTEL Collectors to further enhance deep diving into your service problems using OpenSearch Dashboards. Below are the key features that we are targeting for next (1.2) Data Prepper release:
+A [roadmap](https://github.com/opensearch-project/data-prepper/projects/1) for Data Prepper was published in the Data Prepper OpenSearch Project repository on GitHub. It will give you visibility on plans for new features, enhancements, and bug fixes. We look forward to your feedback on it. In the next few months, the focus is on log collection from multiple sources through Data Prepper where they can be ingested, viewed, and analyzed in OpenSearch Dashboards. Data Prepper 1.2 (December 2021) release is going to provide users the ability to send logs from Fluent Bit to OpenSearch or Amazon OpenSearch Service and use Grok to enhance the logs. These logs can then be correlated to traces coming from the OTEL Collectors to further enhance deep diving into your service problems using OpenSearch Dashboards. Below are the key features that we are targeting for next (1.2) Data Prepper release:
```

Comment:
How do we want to update this? I don't think we want to update this to our current (2022/2023) plans. Otherwise we will need to keep updating this paragraph for each release. That is what the roadmap linked to provides.

My recommendation is that we leave this section mostly the way it is. We could clarify some of the timeframes so that it is clearer to readers that this was about a specific point in time.

For example:

> In the next few months, the focus is on log collection

This could become:

> In late 2021, the focus is on log collection

And this one:

> targeting for next (1.2) Data Prepper release

Could become:

> targeting for the Data Prepper 1.2 release

Reply:
Would it be possible to add a link to the Data Prepper 1.2 release blog here: https://opensearch.org/blog/technical-post/2021/12/Introducing-Data-Prepper-1.2.0-with-Log-Pipelines/

Reply:
Added link at fifth sentence.